makefile.shade

```diff
@@ -70,7 +71,67 @@ var buildTarget = "compile"
 
 #verify-all .pull .change-default-build-target-to-verify .build-all
 
-#build-and-install-all .change-default-build-target-for-coherence-build .build-all
+#ci-build
+    @{
+        var ciVolatileShare = Environment.GetEnvironmentVariable("CI_VOLATILE_SHARE");
+        var nugetExe = Environment.GetEnvironmentVariable("PUSH_NUGET_EXE");
+        var universeArtifacts = Path.Combine("artifacts", "build");
+        Directory.CreateDirectory(universeArtifacts);
+        buildTarget = Environment.GetEnvironmentVariable("KOREBUILD_BUILD_TARGETS") ?? "--quiet compile nuget-install";
+
+        foreach (var batch in BatchReporties())
+        {
+            Parallel.ForEach(batch, repoInfo =>
+            {
+                var repo = repoInfo.Name;
+                var blockName = string.Format("Building {0}", repo);
+                if (IsTeamCity)
+                {
+                    Log.Info(string.Format("##teamcity[blockOpened name='{0}']", FormatForTeamCity(blockName)));
```

Comment:
It automatically reports targets, but I believe @pranavkm  is trying to explicitly open a new block.

Btw @pranavkm, you can reuse some of that code. (this is c# 6, so you may have to adjust for mono). See https://github.com/aspnet/KoreBuild/blob/dev/build/shade/_use-ci-loggers.shade#L4

```c#
(Log as IBlockLogger)?.StartBlock(blockName);
(Log as IBlockLogger)?.EndBlock(blockName);
```